themes/default/content/blog/pulumi-go-boilerplate-v2/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: The Easier Way to Create Pulumi Providers in Go
+allow_long_title: true
+date: 2023-01-19
+authors: 
+  - kyle-dixler
+tags:
+  - go
+  - packages
+meta_desc: A major update to the Pulumi Provider Boilerplate simplifies the provider development process.
+meta_image: meta.png
+---
+
+We are excited to announce that we've updated the Pulumi Provider Boilerplate to make custom provider implementation easier!
+
+**Note:** This update will not have any impact on existing providers that are utilizing an older version of the repository.
+
+This major update brings a wealth of usability improvements to the Pulumi Provider Boilerplate by incorporating our brand
+new Pulumi Go Provider SDK.
+
+<!--more-->
```

Comment:
@cnunciato how about

```suggestion
We are excited to announce that we've updated the Pulumi Provider Boilerplate to make custom provider implementation easier!

This major update brings a wealth of usability improvements to the Pulumi Provider Boilerplate by incorporating our brand
new Pulumi Go Provider SDK.

<!--more-->

**Note:** This update will not have any impact on existing providers that are utilizing an older version of the repository.
```

Reply:
Perfect! (Although I'd love it even more without those hard line breaks, but I can live with it. 😄)